dot/telemetry/mailer_test.go

```diff
@@ -37,7 +38,9 @@ func TestMain(m *testing.M) {
 		Endpoint:  "ws://127.0.0.1:8001/",
 		Verbosity: 0,
 	}
-	GetInstance().AddConnections(append(testEndpoints, testEndpoint1))
+
+	logger := log.New(log.SetLevel(log.DoNotChange))
```

Comment:
Ehh I'm not sure if `DoNotChange` is appropriate here :thinking: It has a specific purpose which I don't exactly remember, it's not really a level.
Also why is it a global logger? Can't you have a logger local to each test i.e. `logger := log.New(log.SetWriter(io.Discard))` You could also have a mocked logger but that's for another day!

Reply:
We need to inject a logger to BootstrapMailer